### ISSUE TITLE
Cannot deserialize when serde attributes are specified

### ISSUE THREAD

**Issue by JoyceBabu** [twistedfall/serde_rusqlite#3]
Does the crate support serde attributes? Deserialization fails when the attributes `alias` and/or `default` is used.

```
#[derive(Clone, Serialize, Deserialize, Debug, PartialEq)]
pub struct Festival {
	festival_id: u32,
	festival_name: String,
	festival_date: String,
	festival_type: u8,
	festival_date_type: u8,
	festival_is_holiday: bool,
}
```

But the following fails

```
#[derive(Clone, Serialize, Deserialize, Debug, PartialEq)]
pub struct Festival {
	#[serde(alias = "festival_id")]
	id: u32,
	#[serde(alias = "festival_name", default = "empty_string")]
	name: String,
	#[serde(alias = "festival_date")]
	date: String,
	#[serde(alias = "festival_type")]
	ftype: u8,
	#[serde(alias = "festival_date_type")]
	date_type: u8,
	#[serde(alias = "festival_is_holiday")]
	is_holiday: bool,
}

pub fn empty_string() -> String {
	String::from("")
}
```

**Comment by twistedfall** [comment]
Hi, I can confirm the issue with at least `alias`. I'll try to release the fixed version asap

**Comment by twistedfall** [comment]
Fixed in 0.22, please note that there are non backwards compatible interface changes: https://github.com/twistedfall/serde_rusqlite/commit/345cc346fa55d622bceb01ad3447546e5b429757
